packages/eslint-plugin/tests/rules/explicit-member-accessibility.test.ts

```diff
@@ -352,7 +352,7 @@ class Test {
 export class XXXX {
   public constructor(readonly value: string) {}
 }
-      `,
+      `.trim(),
```

Comment:
Even when I merge that change into my branch, I get a lot of errors like this:

```

    Expected value to strictly be equal to:
      "
    class Test {
      public constructor(public x: number) {}
    }
                  "
    Received:
      "
    class Test {
      public constructor(public x: number) {}
    }
          "
```

This happens because the `suggestions` expected output is another few levels indented from where the `code` is specified.

Is there something else I should be doing to make it ignore these kinds of whitespace differences?

Reply:
Oh, I see that I no longer need the trailing `` ` `` to be in the same column.